web/src/components/planner/PlannerTooManyFactoriesOpen.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <div v-if="showAlert" class="count-warning bg-amber w-100 text-center py-1 position-absolute">
+    <span v-if="tooManyShownFactories" class="mr-2"><b>Too many factories are visible (>{{ factoryLimit }})!</b> Please hide some factories or your planner will MAJORLY lag!</span>
+    <span v-if="tooManyInputs" class="mr-2"><b>Too many imports are being shown (>{{ inputLimit }})</b>, which causes large amounts of lag. Please hide some factories!</span>
+    <v-btn density="compact" @click="emit('hide-all')">Hide all</v-btn>
+  </div>
+</template>
```

Comment:
Might be a nice touch to add a close button to this warning for any "power users" that want to keep everything open despite slower performance.

Reply:
Yeah I might put it in later on :)